app/ide-desktop/eslint.config.js

```diff
@@ -87,10 +87,6 @@ const RESTRICTED_SYNTAXES = [
         selector: `:matches(ImportDefaultSpecifier[local.name=/^${NAME}/i], ImportNamespaceSpecifier > Identifier[name=/^${NAME}/i])`,
         message: `Don't prefix modules with \`${NAME}\``,
     },
-    {
-        selector: 'TSTypeLiteral',
-        message: 'No object types - use interfaces instead',
```

Comment:
not sure it's worth removing this rule just for a single type

Reply:
I actively use inline object types there(e.g. https://github.com/enso-org/enso/pull/10202/files#diff-da2aef9f2124ef55356bc2554513a6034b842304d0a73f7efa8c872f4e252ab7R104), extracting these types into separate interfaces is tedious

Reply:
they're not supposed to be separate interfaces, this is what `extends` is for in interfaces.